src/parcsr_ls/par_ilu_setup.c

```diff
@@ -7,6 +7,7 @@
 #include "_hypre_parcsr_ls.h"
 #include "_hypre_utilities.hpp"
 #include "par_ilu.h"
+#include "seq_mv.hpp"
```

Comment:
I would like the `.hpp` header file inclusion to parallel the `.h` header inclusion, so ideally we should have an include of a single `_hypre_parcsr_ls.hpp` file here.  The hypre team can sort this out later, though, in a separate PR.

Reply:
Hi @rfalgout , `seq_mv.h` is included in `_hypre_parcsr_mv.h` which is included in `_hypre_parcsr_ls.h`. Since we have `seq_mv.hpp` now, shall we have `_hypre_parcsr_mv.hpp` and then `_hypre_parcsr_ls.hpp`?

Reply:
Yes, that's the idea, but it's not essential that we do that in this PR.  Your call.  Thanks!

Reply:
@rfalgout Thank you for confirming it.